src/main/java/edu/kis/powp/jobs2d/command/OperateToCommand.java

```diff
@@ -39,15 +39,13 @@ public int getPosX() {
 		return posX;
 	}
 
-	public void setPosX(int posX) {
-		this.posX = posX;
-	}
-
 	public int getPosY() {
 		return posY;
 	}
 
-	public void setPosY(int posY) {
-		this.posY = posY;
+	@Override
+	public void replace(Object o) {
+		this.posX = ((OperateToCommand) o).getPosX();
+		this.posY = ((OperateToCommand) o).getPosY();
 	}
```

Comment:
My suggestion concerned mutability. This solution is still mutable but is even worse. Casting is a bad practice and asmell of bad design. So either revert to setters or try to avoid mutability (create new command instead of modifying).